libraries.json

```diff
@@ -8,5 +8,15 @@
         "lp-access-token": "LP_ACCESS_TOKEN",
         "lp-access-secret": "LP_SECRET_TOKEN",
         "library-regex": "spark-*.jar"
+    },
+    {
+        "name": "hadoop",
+        "lp-building-project": "soss",
+        "lp-building-repo": "~data-platform/soss/+source/hadoop/+git/hadoop/",
+        "lp-building-branch-prefix": "lp-3.3.6",
+        "lp-consumer-key": "LP_CONSUMER_KEY",
+        "lp-access-token": "LP_ACCESS_TOKEN",
+        "lp-access-secret": "LP_SECRET_TOKEN",
+        "library-regex": "hadoop-*ubuntu*.jar"
```

Comment:
Not sure the regex is sensible. Issuing the command below in the folder where the tar ball is extracted provides

```
find . -name "hadoop-*ubuntu*.jar"
```

Provides this file 
[result.txt](https://github.com/user-attachments/files/15536094/result.txt)

Reply:
Do you have any thought @welpaolo ?